packages/models/src/lib/content/content.model.ts

```diff
@@ -1,13 +1,18 @@
 import { UserInfo } from '../users';
 import { PubStatus } from './pub-status.enum';
 import { ContentKind } from './content-kind.enum';
+import { ContentRating } from '../works';
 
 export interface ContentModel {
     readonly _id: string;
     readonly author: string | UserInfo;
     title: string;
     desc: string;
     body: string;
+    meta: {
+        rating: ContentRating;
+        warnings: string[];
```

Comment:
should this be an array of  `ContentWarning`s? (yes I'm aware that'll end up being an array of strings anyway)